_posts/2024-03-29-geospatial-techno.md

```diff
@@ -0,0 +1,78 @@
+
+The **Legend** area allows you to preview the legend for the style. Click on the **Preview legend** link to generate a legend based on the current settings.
+
+At the bottom of the Style Editor page, you'll find several options: **Validate**, **Apply**, **Save** and **Cancel**. During editing and especially after editing is complete, you can check the validation of the syntax by pressing the **Validate** button at the bottom. If any validation errors are found, they will be displayed a red message and if no errors are found, you will see the green message. To make changes, press the **Apply** button to access all the tabs and finally press the **Save** button.
+
+After having created the style, it's time to apply it to the layer. To do it, follow these steps:
+- Navigate to the **Data > Layers** page then click on the **layer's name** link to open the layer's properties form. Switch to the Publishing tab.
+- Go to the **Style** section and from the **Default Style list**, select the suitable style, then press the **Save** button.
+- Navigate to the **Data > Layer Preview ** page and open up OpenLayers preview for the layer.
+
+## Edit a Style
+On the Styles page, click on the style name to open the **Style Editor**. The Style Editor page presents the style definition and contains four tabs with many configuration options: **Data** , **Publishing** , **Layer Preview** and **Layer Attributes**.
+- Data tab: The Data tab includes basic style information, the ability to generate a style, and legend details. Moreover, it allows for direct editing of style definitions at the bottom, with support for line numbering, automatic indentation, and real-time syntax highlighting. Users can switch between tabs to create and edit styles easily and can adjust the font size of the editor.
+- Publishing tab: This tab shows all layers available on the server, along with their default style and any additional styles they may have. Users can easily see which layers are linked to the current style by checking a box in the table.
+- Layer Preview tab: This tab enables users to preview and edit the current style of any layer without switching pages. Users can easily select the desired layer to preview and fine-tune styles to continuously test visualization changes.
+- Layer Attributes tab: The Layer Attributes tab shows a list of attributes for the selected layer, making it easy to see and work with the attributes associated with the layer. This can help in deciding which attribute to use for labeling or setting up scale-dependent rules. 
+
+## Creating SLD styles by QGIS
+QGIS has a style editor for map rendering with various possibilities, including the export of raster styles to SLD for use in GeoServer. For versions before 3.4.5, a plugin called **SLD4raster** is required for exporting SLD for use in GeoServer.
+
+Here is a step by step guide to style a raster layer for GeoServer:
```

Comment:
What about vector styles?

Reply:
The video focuses only on creating styles for raster in QGIS.

Reply:
It would be better to at least identify the possibility.

Reply:
It's done now